MiniApp/Classes/core/Display/Displayer.swift

```diff
@@ -21,15 +22,17 @@ internal class Displayer {
           adsDisplayer: adsDisplayer,
           displayNavBar: navConfig?.navigationBarVisibility ?? .never,
           navigationDelegate: navConfig?.navigationDelegate,
-          navigationView: navConfig?.navigationView)
+          navigationView: navConfig?.navigationView,
+            analyticsConfig: analyticsConfig)
```

Comment:
looks like a bit formation need e.g. removing extra space from beginning.

Reply:
The editor is formatting that way 😢